### PR TITLE
Ingest logs for blob containers

### LINKED ISSUE #2309: Add blob storage logs
Part of #1922

### PR DESCRIPTION
**Pull request by JimMadge**
## :white_check_mark: Checklist

- [x] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [x] You are targeting the appropriate branch. If you're not certain which one this is, it should be **`develop`**.
- [x] Your branch is up-to-date with the **target branch** (it probably was when you started, but it may have changed since then).

### :closed_umbrella: Related issues

Closes #2309 
Contributes to #1922 

### :microscope: Tests

Tested on a new SRE deployment

Screenshot 2024-11-28 at 13 45 01

### CODE REVIEW

**Comment by craddm** [comment]
I did actually make a start on this earlier https://github.com/craddm/data-safe-haven/tree/add-storage-logging - but didn't get very far before having to work on something else

File: docs/source/management/logs.md [hunk 1]
```diff
@@ -11,6 +11,38 @@ Select which Log Analytics Workspace you want to view by clicking on the workspa
 
 The logs can be filtered using [Kusto Query Language (KQL)](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-query-overview).
```
Comment (jemrobinson) [inline docs/source/management/logs.md:12]: For the future - we should add some useful KQL queries to this repo and/or the docs

Comment (JimMadge) [reply docs/source/management/logs.md:12]: Looks like we can build queries and deploy them with [pulumi azure native](https://www.pulumi.com/registry/packages/azure-native/api-docs/operationalinsights/query/) :tada:.

Comment (JimMadge) [reply docs/source/management/logs.md:12]: Issue [here](https://github.com/alan-turing-institute/data-safe-haven/issues/2322)

**Review by craddm — APPROVED** [review]
LGTM

**Pull request merged**